libs/category/state/src/category-state.module.ts

```diff
@@ -16,4 +25,14 @@ import { DAFF_CATEGORY_STORE_FEATURE_KEY } from './reducers/public_api';
     DaffProductStateModule,
   ],
 })
-export class DaffCategoryStateModule { }
+export class DaffCategoryStateModule {
+  static forRoot(config: DaffCategoryStateConfiguration = defaultDaffCategoryStateConfiguration): ModuleWithProviders<DaffCategoryStateModule> {
+    return {
+      ngModule: DaffCategoryStateModule,
+      providers: [
+        { provide: DaffCategoryStateConfigurationToken, useValue: config },
+        { provide: DaffDefaultCategoryPageSize, useValue: config.defaultPageSize },
```

Comment:
If this is provided here, I don't think `@daffodil/category/routing` should be its own subpackage

Reply:
@griest024 the routing module's were your idea, what do you think?

Reply:
I don't think it makes sense for guards and resolvers to be in a `state` package. I am still in favor of the `routing` subpackage, despite what the dependency graph looks like.

Reply:
so, what I'm saying is that it doesn't make sense to provide the default page size here if we're going to have a routing subpackage. If it doesn't make sense for guards and resolvers to be in a state package, then it also shouldn't make sense to provide the default configuration for the routing subpackage in the state package.

Reply:
I don't think this token is limited solely to the resolvers/guards.

Reply:
oh, I hadn't noticed. That makes sense